dali/operators/generic/slice/slice_attr.cc

```diff
@@ -61,7 +61,7 @@ is incompatible with providing positional inputs anchor and shape.)code",
 Providing named arguments ``start``, ``end``, ``shape``, ``rel_start``, ``rel_end``, ``rel_shape``
 is incompatible with providing positional inputs anchor and shape.)code",
         nullptr, true)
-    .AddOptionalArg<std::vector<int>>("rel_shape",
+    .AddOptionalArg<std::vector<float>>("rel_shape",
```

Comment:
Can you add a test for `rel_shape`?

Reply:
There is a test, but this only throws an error when you use the object API (not the functional one). I wonder what to do about it.

Reply:
I found that the test didn't check for python list arguments, so I extended (I verified the new tests failed before this fix)